sweep/fee_bumper.go

```diff
@@ -629,3 +873,22 @@ func calcCurrentConfTarget(currentHeight, deadline int32) uint32 {
 
 	return confTarget
 }
+
+// isConfirmed is a helper method that returns a boolean indicating whether the
+// tx is confirmed.
+func isConfirmed(event *chainntnfs.ConfirmationEvent) bool {
+	select {
+	// TODO(yy): check if there's a race between sending a block epoch and
+	// notifying a confirmed tx. If the block epoch is sent first, and the
+	// tx is confirmed in this block, by the time we check the confirmation
+	// event, it might not be sent yet.
+	case event := <-event.Confirmed:
```

Comment:
This function is not idempotent as best I can tell, right? Do we want to *make it idempotent* by replacing the value in the channel? Is the channel receive only?

Reply:
yeah turns out we cannot do this due to the reason listed above.